<ui version="4.0" >
 <class>userWindow</class>
 <widget class="QMainWindow" name="userWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Kullanıcı Ayarları - Anatolya Kurulum</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" >#userWindow { 
	background-image: url(:/artalan/artalan.png); }</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="1" column="0" colspan="3" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string/>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="1" column="1" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Kullanıcı Ayarları</string>
         </property>
         <widget class="QLineEdit" name="userPasswdLine_2" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>310</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>30</y>
            <width>81</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Root Parolası:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>80</y>
            <width>56</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Tekrar:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>160</y>
            <width>131</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Kullanıcı Oluştur:&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
         <widget class="QLineEdit" name="rootPasswdLine_1" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>50</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_4" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>240</y>
            <width>56</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Şifre:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="rootPasswdLine_2" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>100</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="userNameLine" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>210</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="userPasswdLine_1" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>260</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>190</y>
            <width>81</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Kullanıcı ismi:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>290</y>
            <width>56</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text" >
           <string>Tekrar:</string>
          </property>
         </widget>
         <widget class="Line" name="line" >
          <property name="geometry" >
           <rect>
            <x>30</x>
            <y>140</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_7" >
          <property name="geometry" >
           <rect>
            <x>230</x>
            <y>40</y>
            <width>271</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#00aaff;">* root (sistem yöneticisi) parolası&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#00aaff;">sisteminizi koruyabilmeniz için gereklidir.&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8" >
          <property name="geometry" >
           <rect>
            <x>240</x>
            <y>200</y>
            <width>191</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#00aaff;">* Yeni bir kullanıcı oluşturunuz.&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#00aaff;">Sisteme root kullanıcısı ile girmeniz kesinlikle&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#00aaff;">önerilmemektedir.&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer name="horizontalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" >
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="nextButton" >
      <property name="styleSheet" >
       <string notr="true" />
      </property>
      <property name="text" >
       <string>İleri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="backButton" >
      <property name="enabled" >
       <bool>false</bool>
      </property>
      <property name="text" >
       <string>Geri</string>
      </property>
      <property name="icon" >
       <iconset resource="../../images/anatolyaimages.qrc" >
        <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../images/anatolyaimages.qrc" />
 </resources>
 <connections/>
</ui>
